### PR TITLE
Add osx-arm64 build

### PR DESCRIPTION
**Pull request by jonashaag**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jonashaag** [comment]
@conda-forge-admin rerender

**Comment by jonashaag** [comment]
@conda-forge-admin rerender

**Comment by jonashaag** [comment]
@conda-forge-admin rerender

**Comment by jonashaag** [comment]
@conda-forge-admin rerender

**Comment by jonashaag** [comment]
@conda-forge-admin rerender

File: recipe/build.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+if [[ "${target_platform}" == "osx-arm64" ]]; then
+  # basic_check_build() does not work when cross compiling
```
Comment (tomasvanpottelbergh) [inline recipe/build.sh:6]: Could you add a reference here to the issue you created in the pmdarima repo?

**Review by tomasvanpottelbergh — CHANGES_REQUESTED** [review]

**Comment by tomasvanpottelbergh** [comment]
Sorry for the late response, but thanks a lot for making this work! Good to merge if you address my small comment.l

**Comment by jonashaag** [comment]
Can you add comment and merge while you’re at it?

**Review by tomasvanpottelbergh — APPROVED** [review]

**Pull request merged**